### PR TITLE
Book detail style

### PR DESCRIPTION
**Pull request by blacksam07**
## Summary
- Add borderline in title
- Add shadows 

## Trello Card

[28-detalle-de-libro-iii-estructura](https://trello.com/c/RWumOerD/28-detalle-de-libro-iii-estructura)

## Screenshots
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,10 +10,12 @@
 </head>
 <body class="container">
   <img class="container-item book-cover" src="resources/book-cover.png" alt="book_cover">
-  <div class="container-item">
-    <h2 class="bold-title">Título del libro
-      <span class="base-span subtitle">(género)</span>
-    </h2>
+  <div class="container-item container-data">
+    <div class="border-line">
```
Comment (CrisVega08) [inline index.html]: Delete this i try other way for line

File: scss/detail.scss [hunk 2]
```diff
@@ -25,17 +25,27 @@
 }
 
 .container {
+  background-color: $white;
+  box-shadow: 4px 4px 10px 0 $black-lighter-5;
   display: flex;
-  justify-content: center;
+  height: 416px;
```
Comment (CrisVega08) [inline scss/detail.scss]:
```suggestion
  height: 415px;
```

File: scss/detail.scss [hunk 3]
```diff
@@ -25,17 +25,27 @@
...
   margin: 24px 26px
 }
 
+.container-data {
+  flex-grow: 2;
+}
+
 .book-cover {
   height: 368px;
   width: 261px;
```
Comment (CrisVega08) [inline scss/detail.scss]:
```suggestion
  width: 260px;
```

File: scss/detail.scss [hunk 4]
```diff
@@ -25,17 +25,27 @@
...
+  background-color: $white;
+  box-shadow: 4px 4px 10px 0 $black-lighter-5;
   display: flex;
-  justify-content: center;
+  height: 416px;
+  width: 900px;
 }
 
 .container-item {
   margin: 24px 26px
```
Comment (CrisVega08) [inline scss/detail.scss]:
```suggestion
  margin: 25px
```

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

**Review by betoobernal — APPROVED** [review]

**Review by CrisVega08 — APPROVED** [review]

**Pull request merged**